website/docs/getting-started/setup/installation-guides/aws-ami.md

```diff
@@ -86,6 +88,10 @@ You see a launch status screen as shown below:
       The application password is **only** available in system logs for the initial 24 hours. It's recommended that create a new admin user and delete the `appsmith@example.com user` immediately.
       :::
 
+ c. For **Business Edition**:
```

Comment:
do business users have to do steps a and b?

Reply:
Yes. and in addition to that, the c. I guess I can rewrite it to Only for Business Edition.

Reply:
Can we write it as 

> If you are on **Business Edition**, complete the following steps:

Reply:
If you are on Business Edition, "complete the following steps" doesn't sound appropriate, as they already follow the steps. For Business Edition Only should be good. As it clearly states that only users on Business edition should follow these steps.

Reply:
Did rewording. Let's discuss if needed.